src/alto-ui/Form/DatePicker/DatePickerHeader.js

```diff
@@ -131,8 +164,8 @@ class DatePickerHeader extends React.Component {
             <ButtonGroup
               className="DatePicker__am-pm-choice"
               items={[{ title: 'am', value: -12 }, { title: 'pm', value: 12 }]}
-              value={date.hour > 11 ? 12 : -12}
-              onChange={delta => this.props.onChangeTime({ hour: date.hour + delta })}
+              value={hour > 11 ? 12 : -12}
```

Comment:
may be we should use constats for `11`, `12`, `-12` ?

Reply:
I don't know what those are.
I didn't check other places of this code and I don't know how to name these.
Sorry, but I'm not competence person to change it now